lib/r2d2/payment_token.rb

```diff
@@ -60,8 +60,7 @@ def decrypt_message(encrypted_data, symmetric_key)
         decipher.decrypt
         decipher.key = symmetric_key
         decipher.auth_data = ""
-        payload = decipher.update(Base64.decode64(encrypted_data)) + decipher.final
-        payload.unpack('U*').collect { |el| el.chr }.join
+        decipher.update(Base64.decode64(encrypted_data)) + decipher.final
```

Comment:
Yep, and it would actually wreak havoc with non-ASCII characters:
```
"pâylöad".unpack('U*').collect { |el| el.chr }.join
#=> "p\xE2yl\xF6ad"
```